src/java.base/share/classes/java/lang/AbstractStringBuilder.java

```diff
@@ -260,7 +275,7 @@ private void ensureCapacityInternal(int minimumCapacity) {
      * @throws OutOfMemoryError if minCapacity is less than zero or
      *         greater than (Integer.MAX_VALUE >> coder)
      */
-    private int newCapacity(int minCapacity) {
+    private static int newCapacity(int minCapacity, byte[] value, byte coder) {
```

Comment:
1.  Only the current length is needed and should be the argument.
2. Add the \@param tags for the new arguments.